scripts/eurostat/health_determinants/physical_activity/process.py

```diff
@@ -0,0 +1,860 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file.
+"""
+from sys import path
+# For import common.replacement_functions
+path.insert(1, '../')
+from common.replacement_functions import (_replace_sex, _replace_physact,
+                                          _replace_isced11, _replace_quant_inc,
+                                          _replace_deg_urb, _replace_levels,
+                                          _replace_duration, _replace_c_birth,
+                                          _replace_citizen, _replace_lev_limit,
+                                          _replace_bmi, _split_column)
+# For import util.alpha2_to_dcid
+path.insert(1, '../../../../')
```

Comment:
Current implementation has limitation to run module from current folder only. Make similar changes as suggested in input_files.py to add modules / packages.